src/controllers/transfers.js

```diff
@@ -65,12 +66,13 @@ exports.getResource = function * fetch () {
  * @apiVersion 1.0.0
  *
  * @apiDescription Use this to get a signed receipt containing only the id of
- *   transfer and its state.
+ *   transfer and its state. It functions even if the transfer doesn't exist yet.
```

Comment:
Once we have a dedicated endpoint for getting the key and signature type that will be fine, but for now we need to be able to get the public_key and signature type before the transfer has been created.